src/main/resources/messages.properties

```diff
@@ -273,16 +252,48 @@ choose-programs.rca.short-desc=Get help for refugees
 choose-programs.rca.desc=<br>Gives cash to people who are refugees until they get jobs and are able to be financially independent.
 choose-programs.other=I want to add another program not shown here
 choose-programs.other.desc=<br>If you believe there are other benefits you can get or a caseworker at an office believes you can get benefits from another program, choose this option.
-# Expedited SNAP
-expedited-snap.title=Expedited notice
-expedited-snap.header=If you are eligible, you can get expedited food assistance (SNAP) in seven days or less
-expedited-snap.what-is=What is expedited SNAP?
-expedited-snap.what-is.content=Expedited SNAP means you can get your food benefits more quickly. People who qualify for expedited SNAP can get their EBT card delivered in 7 days or less.
-expedited-snap.who-qualifies=Who qualifies for expedited SNAP?
-expedited-snap.who-qualifies.content=You may qualify for expedited SNAP if any of these situations are true:
-expedited-snap.who-qualifies.li1=Your monthly income is less than $150 and you have $100 or less in personal savings, bank accounts, or cash on hand.
-expedited-snap.who-qualifies.li2=The amount you pay for rent or mortgage, and utilities is more than your income and savings.
-expedited-snap.who-qualifies.li3=You are a migrant or seasonal farmworker who is not working right now and has $100 or less in personal savings, bank accounts, or cash on hand.
+# Expedited SNAP Notice
+expedited-snap-notice.title=Expedited SNAP notice
+expedited-snap-notice.header=We see you want help with food.
+expedited-snap-notice.no-income=If you have no income or very low income, we check to see if you can get benefits in 7 days or less.
+expedited-snap-notice.expedited-snap=This is called expedited SNAP.
+# OHEP Notice
+ohep-notice.title=OHEP Notice
+ohep-notice.header=When you apply for SNAP or TCA, we will check to see if you can get help with heating electric and gas bills.
+ohep-notice.ohep=How does this work?
+ohep-notice.ohep-l1=A recent law change says that people who are able to get SNAP or TCA benefits are also eligible for help through the Office of Home Energy Programs (OHEP).
+ohep-notice.ohep-l2=If you are eligible, Maryland will pay the company that sends you bills directly.
```

Comment:
This wording does not match the design. Did the design change after you wrote these pages?